src/components/Dashboard/UserMenu.jsx

```diff
@@ -127,7 +130,7 @@ export default class UserMenu extends Component {
                 {profile.displayName[0]}
               </Avatar>
             )}
-            <ListItemText primary={profile.displayName} style={usernameStyle} />
+            <ListItemText primary={profile.displayName} primaryTypographyProps={{className:classes.usernameStyle}} />
```

Comment:
Travis is complaining because of linting errors. You can fix those straight from the terminal by running `yarn lint --fix`.